src/Rule/Nested.php

```diff
@@ -220,7 +219,7 @@ final class Nested implements
      * @psalm-param WhenType $when
      */
     public function __construct(
-        iterable|object|string|null $rules = null,
+        iterable|object|string $rules = [],
```

Comment:
`null` need for use attributes from attributes of validated object

Reply:
Because `RulesProviderInterface::getRules()` return type doesn't allow `null` values:

https://github.com/yiisoft/validator/blob/db5d9e3564ec30e5b8c0cc9d7963a721f79c31d1/src/RulesProviderInterface.php#L15

The handler is modified accordingly.

Reply:
Means `Nested` must not implement `RulesProviderInterface`.